tests/test_cli_claude_teams_fallback_path.py

```
@@ -74,7 +98,111 @@ def main() -> int:
+        with focused_cmux_server(tmp / "focused-cmux.sock") as (socket_path, requests):
+            focused_env = env.copy()
+            focused_env["CMUX_SOCKET_PATH"] = socket_path
+            for key in (
+                "CMUX_WORKSPACE_ID",
+                "CMUX_SURFACE_ID",
+                "CMUX_PANEL_ID",
+                "CMUX_TAB_ID",
+                "CMUX_PANE_ID",
+            ):
+                focused_env.pop(key, None)
+            focused = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)